keyboards/quarkeys/z40/readme.md

```diff
@@ -0,0 +1,21 @@
+# Quarkeys Z40 Low Profile Keyboard
+
+A 40% Bottom mount low-profile & MX switch compatible keyboard.
+
+* Keyboard Maintainer: [Quarkeys Studio](www.quarkeys.com)
+* Hardware Supported: Z40-ACR, Z40-Pastry
+* Hardware Specs: Per-key RGBs, layer & Capslock indicators
+* For [RGB Matrix feature](https://docs.qmk.fm/#/feature_rgb_matrix): `RGB_MATRIX_ENABLE = yes`, `RGB_MATRIX_DRIVER = WS2812`, `RGBLIGHT_ENABLE = no`.
```

Comment:
No need to set the driver since it's already set. 
```suggestion
* For [RGB Matrix feature](https://docs.qmk.fm/#/feature_rgb_matrix): `RGB_MATRIX_ENABLE = yes`, `RGBLIGHT_ENABLE = no`.
```

Reply:
@drashna thx, so, is it ok to leave rgb matrix driver setup in rules.mk even in rgblight mode?

Reply:
Yes, absolutely.  Since rgb matrix isn't enabled, the driver setting isn't evaluated.  So there is no harm in having it set.

Reply:
@drashna thank you so much. I tested it, it works fine, but your explanation clears the fog.